pkg/daemon/reader.go

```diff
@@ -301,6 +315,16 @@ func readerPollLoop(
 
 		newScanned, removed, err := pollDevice(cfg, &pnd, activeCard, ttp, pbp)
 
+		if removed {
+			st.SetCardRemovalTime(time.Now())
+			candidateForRemove = true
+			// we return a reference to activeCard we didn't change card
+		} else if (newScanned !== activeCard) {
+			// card is changed
+			st.SetCardRemovalTime(0)
```

Comment:
@wizzomafizzo is this enough to be sure that the condition `time.isZero()` will work? Or how do i set a time to the zero moment?